src/main/java/de/drachir000/utils/config/Configuration.java

```diff
@@ -182,6 +182,70 @@ public <E extends Enum<E>> E getEnum(Class<E> clazz, String key) throws JSONExce
 		return content.getEnum(clazz, key);
 	}
 	
+	/**
+	 * Get the {@link Byte} value associated with a key.
+	 *
+	 * @param key A key string.
+	 * @return The {@link Byte} value.
+	 * @throws JSONException if the key is not found or if the value is not a {@link Byte}.
+	 * @see Configuration#getOrDefault(String, byte)
+	 */
+	public byte getByte(String key) throws JSONException {
+		Object object = get(key);
+		if (object instanceof Number) {
+			return ((Number) object).byteValue();
+		}
+		try {
+			return Byte.parseByte(object.toString());
+		} catch (Exception e) {
+			throw new JSONException("JSONObject[" + JSONObject.quote(key) + "] is not a " + "byte" + " (" + object.getClass() + " : " + object + ").", e);
+		}
+	}
+	
+	/**
+	 * Get the {@link Short} value associated with a key.
+	 *
+	 * @param key A key string.
+	 * @return The {@link Short} value.
+	 * @throws JSONException if the key is not found or if the value is not a {@link Short}.
+	 * @see Configuration#getOrDefault(String, short)
+	 */
+	public short getShort(String key) throws JSONException {
+		Object object = get(key);
+		if (object instanceof Number) {
+			return ((Number) object).shortValue();
+		}
+		try {
+			return Short.parseShort(object.toString());
+		} catch (Exception e) {
+			throw new JSONException("JSONObject[" + JSONObject.quote(key) + "] is not a " + "short" + " (" + object.getClass() + " : " + object + ").", e);
+		}
+	}
+	
+	/**
+	 * Get the {@link Character} value associated with a key.
+	 *
+	 * @param key A key string.
+	 * @return The {@link Character} value.
+	 * @throws JSONException if the key is not found or if the value is not a {@link Character}.
+	 * @see Configuration#getOrDefault(String, char)
+	 */
+	public char getChar(String key) throws JSONException {
+		Object object = get(key);
+		if (object instanceof Character) {
+			return (char) object;
+		}
+		String s = object.toString();
+		if (s.length() > 1) {
+			throw new JSONException("JSONObject[" + JSONObject.quote(key) + "] is not a " + "char" + " (" + object.getClass() + " : " + object + ").");
+		}
+		try {
+			return s.charAt(0);
+		} catch (Exception e) {
+			throw new JSONException("JSONObject[" + JSONObject.quote(key) + "] is not a " + "short" + " (" + object.getClass() + " : " + object + ").", e);
```

Comment:
Theres still a "short" in the Character method here

Reply:
Fixed in https://github.com/Drachir000/SimpleConfigLib/pull/11/commits/92e9ad1d1455908dfb06eb04c6c26a8fd422af4f